static/css/footer.less

```diff
@@ -100,9 +100,9 @@
 
         &-label {
             display: block;
+            font-size: unit(16px / @base-font-size-px, em);
             margin-bottom: unit(8px / @base-font-size-px, em);
             .webfont-demi();
-
```

Comment:
I would say leave this line break since the `respond-to` mixin is similar to nesting.